projects/ad9081/Makefile

```diff
@@ -0,0 +1,15 @@
+TARGET := ad9081
+IIO_SUPPORT = y
```

Comment:
For other projects `TINYIIOD = y` variable is used, so that the 
https://github.com/analogdevicesinc/no-OS/blob/master/tools/scripts/linux.mk
builds libtinyiiod and links with it. Do we want  another variable to be introduced for, iio support?
Does `make run` works for this project? It seems to me that debug will work, since all the sources appear to be copied.

Reply:
I'll declare TINYIIOD instead.